clai/README.md

```diff
@@ -25,57 +25,103 @@ export OPENAI_API_KEY='your-api-key-here'
 Then with [`uvx`](https://docs.astral.sh/uv/guides/tools/), run:
 
 ```bash
-uvx clai
+uvx clai chat
```

Comment:
Hmm can we keep `clai` working as it did, and not require this new argument?

Reply:
we can, I though this would be cleaner/less confusing